MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -77,6 +82,39 @@ public class NavigationDirections: Directions, OfflineRoutingProtocol {
         }
     }
     
+    public class func unpackTilePack(at filePath: URL, outputDirectory: URL, progressHandler: UnpackProgressHandler?, completionHandler: UnpackCompletionHandler?) {
+        
+        OfflineDirectionsConstants.offlineSerialQueue.sync {
+            
+            let totalPackedBytes = filePath.fileSize()!
+            
+            // Report 0% progress
+            progressHandler?(totalPackedBytes, totalPackedBytes)
+
+            var timer: CountdownTimer? = CountdownTimer(countdown: .seconds(500), accuracy: .seconds(500), executingOn: OfflineDirectionsConstants.unpackSerialQueue, payload: {
+                let remainingBytes = filePath.fileSize()!
+                progressHandler?(totalPackedBytes, remainingBytes)
+            })
+
+            timer?.arm()
+            
+            let tilePath = filePath.absoluteString.replacingOccurrences(of: "file://", with: "")
```

Comment:
Constantize: `fileSchemaIdentifier`?

Reply:
Even better, use `URL.path` to omit the schema.